app/api/__init__.py

```diff
@@ -112,3 +115,10 @@
           '/event_copyright/<int:id>', '/events/<int:event_id>/event_copyright')
 api.route(EventCopyrightRelationship, 'copyright_event',
           '/event_copyright/<int:id>/relationships/event')
+
+# event invoices
+api.route(EventInvoiceList, 'event_invoice_list', '/event-invoices', '/events/<int:event_id>/event_invoices',
+          '/users/<int:user_id>/event_invoices')
+api.route(EventInvoiceDetail, 'event_invoice_detail', '/event_invoices/<int:id>')
+api.route(EventInvoiceRelationship, 'event_invoice_user', '/event_invoices/<int:id>/relationships/user')
+api.route(EventInvoiceRelationship, 'event_invoice_event', '/event_invoices/<int:id>/relationships/event')
```

Comment:
@enigmaeth add a few fields too for checking, like id and code or something in discount api. check if you get the correct relationship response